video_ingest_tool/extractors.py

```
@@ -0,0 +1,180 @@
+"""
+Metadata extractors for the video ingest tool.
+
+Contains functions for extracting metadata from video files using various tools.
+"""
+
+import os
+import av
+import pymediainfo
+import exiftool
```

Ruff (0.11.9)

`exiftool` imported but unused

Remove unused import: `exiftool`

(F401)